administrator/language/de-DE/com_joomlaupdate.ini

```diff
@@ -16,26 +16,45 @@ COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_NEXT="Joomla! Next"
 COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_TESTING="Test"
 COM_JOOMLAUPDATE_CONFIGURATION="Joomla-Update: Optionen"
 COM_JOOMLAUPDATE_CONFIRM="Bestätigen"
+COM_JOOMLAUPDATE_EMPTYSTATE_APPEND="Aktualisieren der Website durch manuelles Hochladen des Update-Pakets."
+COM_JOOMLAUPDATE_EMPTYSTATE_BUTTON_ADD="Auf Updates prüfen"
+COM_JOOMLAUPDATE_EMPTYSTATE_CONTENT="Die Schaltfläche unten klicken, um nach Updates zu suchen."
+COM_JOOMLAUPDATE_EMPTYSTATE_TITLE="Überprüfen, ob ein Update verfügbar ist."
 COM_JOOMLAUPDATE_FAILED_TO_CHECK_UPDATES="Fehler beim Überprüfen auf Updates."
-COM_JOOMLAUPDATE_LOGIN="Anmelden"
 COM_JOOMLAUPDATE_MINIMUM_STABILITY_ALPHA="Alpha"
 COM_JOOMLAUPDATE_MINIMUM_STABILITY_BETA="Beta"
 COM_JOOMLAUPDATE_MINIMUM_STABILITY_DESC="Die Mindeststabilität der Updates, die angezeigt werden sollen. „Entwicklung“ ist die Einstellung für die am wenigsten stabilen Versionen. „Stabil“ kann produktiv eingesetzt werden. Wenn der Entwickler keine Stabilität angegeben hat, wird die Version als „Stabil“ angesehen."
 COM_JOOMLAUPDATE_MINIMUM_STABILITY_DEV="Entwicklung"
 COM_JOOMLAUPDATE_MINIMUM_STABILITY_LABEL="Mindeststabilität der Updates"
 COM_JOOMLAUPDATE_MINIMUM_STABILITY_RC="Release Candidate (RC)"
 COM_JOOMLAUPDATE_MINIMUM_STABILITY_STABLE="Stabil"
+COM_JOOMLAUPDATE_NODOWNLOAD_EMPTYSTATE_APPEND="Die Website durch manuelles Hochladen des Update-Pakets aktualisieren."
+COM_JOOMLAUPDATE_NODOWNLOAD_EMPTYSTATE_BUTTON_ADD="Erneute Prüfung auf Update"
+COM_JOOMLAUPDATE_NODOWNLOAD_EMPTYSTATE_CONTENT="Es wurde ein Update für Joomla %1$s gefunden, aber es war nicht möglich, die Download-URL für das Update zu ermitteln. Entweder ist das Update auf Joomla %1$s für diese Stabilitätsstufe nicht verfügbar oder es gibt ein Problem mit dem Joomla Update Server.<br>Bitte das Update-Paket von <a href=\"https://downloads.joomla.org/latest\">der offiziellen Joomla Download-Seite</a> herunterladen und die Upload- und Update-Funktion verwenden."
+COM_JOOMLAUPDATE_NODOWNLOAD_EMPTYSTATE_TITLE="Es kann keine Download-URL gefunden werden"
 COM_JOOMLAUPDATE_OVERVIEW="Joomla-Update"
 COM_JOOMLAUPDATE_PREUPDATE_CHECK_CAPTION="Tabelle der Servereinstellungen, die vor dem Update überprüft werden sollten."
-COM_JOOMLAUPDATE_PREUPDATE_CHECK_COMPLETED_YOU_HAVE_DANGEROUS_PLUGINS="Es sind Plugins installiert und aktiviert, die das Joomla-Upgrade stören und zu einer fehlerhaften Aktualisierung führen können, wodurch die Website nicht mehr zugänglich wäre.<br><br>Es wird dringend empfohlen, diese Plugins vor dem Upgrade zu aktualisieren, zu deaktivieren oder zu deinstallieren."
+COM_JOOMLAUPDATE_PREUPDATE_CHECK_COMPLETED_YOU_HAVE_DANGEROUS_PLUGINS="Es sind Plugins installiert und aktiviert, die das Joomla-Update stören und zu einer fehlerhaften Aktualisierung führen können, wodurch die Website nicht mehr zugänglich wäre.<br><br>Es wird dringend empfohlen, diese Plugins vor dem Update zu aktualisieren, zu deaktivieren oder zu deinstallieren."
```

Comment:
@zero-24 Ist es ein Upgrade?

Reply:
Im Core wird immer zu "Update" gewechselt

Reply:
In Joomla 4 wird der string ja auch bei Updates angezeigt, Update ist ok in meinen augen